File: C++/1662. Check If Two String Arrays are Equivalent.cpp
class Solution {
public:
    bool arrayStringsAreEqual(vector<string>& w1, vector<string>& w2) {
        string a,b;
        int i;
        for(i=0;i<w1.size();i++)
        {
            a+=w1[i];
        }
        for(i=0;i<w2.size();i++)
        {
            b+=w2[i];
        }
        return(a==b);
    }
};

File: JAVA/1662. Check If Two String Arrays are Equivalent.java
class Solution {
    public boolean arrayStringsAreEqual(String[] w1, String[] w2) {
        String a="",b="";
        for(int i=0;i<w1.length;i++)
        {
            a=a+w1[i];
        }
        for(int i=0;i<w2.length;i++)
        {
            b=b+w2[i];
        }
        return(a.equals(b));
    }
}